docs/Clearwater_Stateful_Statistics.md

```diff
@@ -0,0 +1,42 @@
+Despite being designed as an inherently stateless system, Clearwater is able to provide a number of deployment wide stateful statistics. This is done through utilisation of our distributed timer store, Chronos. As components use timers to maintain stateful functionality without actually remaining independently aware of state, we can use those same timers to convey statistics. Currently, only Sprout and Ralf expose any stateful statistics.
```

Comment:
I'm not sure about the wording here - it implies that the stateless design is a bad thing